imagebuildah/stage_executor.go

```diff
@@ -668,6 +668,17 @@ func (s *StageExecutor) prepare(ctx context.Context, from string, initializeIBCo
 		for _, p := range builder.Ports() {
 			ports[docker.Port(p)] = struct{}{}
 		}
+		buildahHealthConfig := builder.Healthcheck()
+		var healthConfig *docker.HealthConfig
+		if buildahHealthConfig != nil {
+			healthConfig = &docker.HealthConfig{
+				Test:        buildahHealthConfig.Test,
+				Interval:    buildahHealthConfig.Interval,
+				Timeout:     buildahHealthConfig.Timeout,
+				StartPeriod: buildahHealthConfig.StartPeriod,
+				Retries:     buildahHealthConfig.Retries,
+			}
+		}
```

Comment:
`Healthcheck()` returns a fresh structure, so we don't really need to make a copy or use a temporary variable here.

Reply:
@nalind They are of different types so had to cast.

Reply:
Ah, right, too many packages named `docker`.